app/renderer/components/navigation/buttons/navigationBarButtonContainer.js

```diff
@@ -17,7 +22,6 @@ class NavigationBarButtonContainer extends React.Component {
         // isNested and isStandalone should not be called at the same time
         this.props.isNested && styles.container_nested,
         this.props.isStandalone && styles.container_standalone,
-        this.props.onNavigationBarChrome && styles.chromeButtonContainer,
```

Comment:
This was reverted on
https://github.com/brave/browser-laptop/pull/9851/commits/80250d37da3f71a372746ce73f84b939aaf0eb7b#diff-857c685f4229536eee936946d457ff72R33
https://github.com/brave/browser-laptop/pull/9851/commits/80250d37da3f71a372746ce73f84b939aaf0eb7b#diff-857c685f4229536eee936946d457ff72R76